lib/command.js

```diff
@@ -2034,7 +2036,9 @@ Expecting one of '${allowedValues.join("', '")}'`);
     }
     context.write(helpInformation);
 
-    this.emit(this._helpLongFlag); // deprecated
+    if (this._helpLongFlag) {
```

Comment:
Skipping the emit is ok, but for consistent behaviour I think do same as other case and use `'--help'` if undefined.

Reply:
Thinking again, I think keep what you have. Skipping the emit when no long flag is fine. There is new better way of appending help so anyone using help option with only a short flag can do it the new way.